src/components/row.js

```diff
@@ -94,7 +94,9 @@ function Row(props) {
         onMouseLeave={() => props.onHighlightState?.()}
         touchstart={() => props.onHighlightState?.(state, props.index)}
         onClick={() => {
-          handleReveal();
+          if (!props.total) {
+            handleReveal();
+          }
         }}
```

Comment:
@rakeshreddy1410  
Can we avoid the attaching of a click event on `Total` as there is no use of it?
Due to your changes click event still will fire after clicking on it only thing  is you are avoiding the call of `handleReveal()` function.

I fill that this is the best way. because there is no use of click event on `Total Bar` so instead of preventing it by adding the condition better don't attach event on `Total bar` 

@shuklaayush  Your opinion also matters.

Reply:
@chinmaydalvi You're right. No point in even attaching a click event to the Total row

Reply:
@chinmaydalvi  An alternate way would be to pass and spoof function to Total in the `Table.js` file.
Like: 
```suggestion
       {states.length > 1 && props.summary === false && (
            <Row key={0} state={states[0]} total={true}  handleReveal={() => false}/>
          )}
```

Reply:
since we are doing conditional validation in row.js which already gets props.total
added the event based on the same condition